plugins/inputs/influxdb_v2_listener/README.md

```diff
@@ -40,6 +40,12 @@ Telegraf minimum version: Telegraf 1.16.0
   ## Optional token to accept for HTTP authentication.
   ## You probably want to make sure you have TLS configured above for this.
   # token = "some-long-shared-secret-token"
+
+  ## Influx parser type to use. Users can choose between 'internal' and
+  ## 'upstream'. The internal parser is what Telegraf has historically used.
+  ## While the upstream parser involved a large re-write to make it more
+  ## memory efficient and performant.
```

Comment:
Maybe simplify this to include only what's needed to make a choice.

```suggestion
  ## InfluxDB line protocol parser.
  ## 'internal' is the default. 'upstream' is a newer parser that is faster
  ## and more memory efficient.
```

Reply:
agreed and done